core/src/main/java/org/elasticsearch/action/DocWriteResponse.java

```diff
@@ -59,6 +59,7 @@
     private static final String _SEQ_NO = "_seq_no";
     private static final String RESULT = "result";
     private static final String FORCED_REFRESH = "forced_refresh";
+    private static final String _INPUT = "_input";
```

Comment:
We've been moving towards just writing the string in quotes at the place where we need it. You can do that if you want. These constants don't buy much.

When we support both parsing and outputting a field we'll declare constants of type `ParseField`.